frame/support/src/traits/misc.rs

```diff
@@ -880,8 +880,9 @@ pub trait GetBacking {
 pub trait EnsureInherentsAreFirst<Block> {
 	/// Ensure the position of inherent is correct, i.e. they are before non-inherents.
 	///
-	/// On error return the index of the inherent with invalid position (counting from 0).
-	fn ensure_inherents_are_first(block: &Block) -> Result<(), u32>;
+	/// On error return the index of the inherent with invalid position (counting from 0). On
```

Comment:
Any specific reason why we need to report the position of the inherent in case of error besides the error message?

Reply:
yes it is only for error message